tests/test_v1.py

```
@@ -4,821 +4,819 @@
+        (day, sec) = j.day_sec_as_type_from_utc((-366,0,366), 0., "TAI")
 
+    with warnings.catch_warnings():
+        warnings.filterwarnings('error', category=JDW)
         # This should yield an error
-        warnings.filterwarnings('error')
-        self.assertRaises(JDW, j.times_in_string, "End time[23:59:60.000]")
+        with pytest.raises(JDW):
+            j.times_in_string("End time[23:59:60.000]")
 
         # This warning was already raised so it should not be repeated
         (day, sec) = j.day_sec_as_type_from_utc((-366,0,366), 0., "TAI")
```

Ruff (0.15.6)

[warning] Unpacked variable `day` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)